core/common/src/main/java/alluxio/TtlExpiryAction.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+/**
+ * Represents an Action to take on file after Ttl.
+ *
+ */
+public enum TtlExpiryAction {
+
+  /**
+   * Indicates that the file has to be deleted (Even in underFS) after Ttl.
+   */
+  DELETE,
+
+  /**
+   * Indicates that the file has to be freed (From Alluxio) after Ttl.
```

Comment:
`Indicates that the file should be freed (i.e. deleted in Alluxio but not in UFS).`